app/controllers/books_controller.rb

```diff
@@ -0,0 +1,13 @@
+class BooksController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
move this to ApplicationController or create an ApiController and inherit of this